<launch>

  <!-- <arg name="ns" default="remote" /> -->
  <arg name="gui" default="false" />
  <!-- <arg name="input_image" default="/$(arg ns)/head_rgbd_sensor/rgb/image_rect_color" /> -->
  <arg name="input_image" default="/kinect_head/rgb/image_color" />
  <arg name="prompt_image" default="/test_pub" />
  <!-- <arg name="prompt_image" default="/test_pub" /> -->
  <!-- <arg name="image" /> -->

  <node name="track_anything"
        pkg="track_anything_ros" type="test_node.py"
        output="screen" clear_params="true" >
    <remap from="~input" to="$(arg input_image)" />
    <!-- <rosparam subst_value="true" > -->
    <!--   score_thresh: 0.2 -->
    <!--   url: $(arg url) -->
    <!-- </rosparam> -->
  </node>


  <node pkg="jsk_rqt_plugins" type="rqt_image_view2" name="$(anon rqt_image_view2)"
        output="screen">
    <remap from="event" to="image_view2/event" />
    <remap from="image_marked" to="image_marked" />
  </node>
  <node pkg="image_view2" type="image_view2" name="image_view2"
        output="screen">
    <remap from="image" to="$(arg input_image)" />
    <!-- <remap from="image" to="$(arg prompt_image)" /> -->
    <param name="use_window" value="false" />
    <!-- <param name="use_window" value="true" /> -->
  </node>

  <node name="track_anything_button"
        pkg="jsk_rqt_plugins" type="rqt_tabbed_buttons" output="screen" clear_params="true">
    <rosparam command="load" file="$(find track_anything_ros)/resource/remote_tabbed_layout.yaml" />
  </node>
  <!-- launch rqt_gui -->
  <!-- <include file="$(find track_anything_ros)/launch/include/rqt_gui.launch" > -->
  <!--   <!-\- <arg name="test_image" value="/kinect_head/rgb/image_color" /> -\-> -->
  <!-- </include> -->


  <!-- <group ns="instance_segmentation" > -->
  <!--   <node name="draw_rects" -->
  <!--         pkg="jsk_perception" type="draw_rects.py"> -->
  <!--     <remap from="~input" to="$(arg input_image)" /> -->
  <!--     <remap from="~input/rects" to="output/rects" /> -->
  <!--     <remap from="~input/class" to="output/class" /> -->
  <!--     <rosparam> -->
  <!--       use_classification_result: true -->
  <!--       show_proba: true -->
  <!--     </rosparam> -->
  <!--   </node> -->

  <!--   <group if="$(arg gui)" > -->
  <!--     <node name="image_view00" -->
  <!--           pkg="image_view" type="image_view" > -->
  <!--       <remap from="image" to="output/viz" /> -->
  <!--     </node> -->
  <!--     <node name="image_view01" -->
  <!--           pkg="image_view" type="image_view" > -->
  <!--       <remap from="image" to="draw_rects/output" /> -->
  <!--     </node> -->
  <!--   </group> -->
  <!-- </group> -->

</launch>
